2001.py

```diff
@@ -0,0 +1,26 @@
+T = int(input())
+
+for test_case in range(1, T+1):
+    a, b = map(int,input().split())
+
+    squ = [list(map(int,input().split())) for _ in range(a)]
+
+    fly = []
+
+    for i in range(a-b+1):
+        for j in range(a-b+1):
+            total = 0
+
+            for ai in range(b):
+                for aj in range(b):
+                    if i + ai in range(a) and j + aj in range(b):
+                        total += squ[i + ai][j + aj]
+
+                fly.append(total)
+
+max = fly[0]
+for i in fly:
+    if i > max:
+        max = i
+
+print('#{} {}'. format(test_case, max))
```

Comment:
깔끔한 풀이네요! 👍
```suggestion
    max = fly[0]
    for i in fly:
        if i > max:
            max = i

    print('#{} {}'. format(test_case, max))
```
모든 테스트 케이스에 대해 출력할 수 있도록 공백을 수정해주셔야 합니다 😄